app/web/features/communities/events/EventTimeChanger.tsx

```diff
@@ -73,36 +73,54 @@ export default function EventTimeChanger({
   const handleStartTimeChange = (e: {
```

Comment:
I've found two minor things while testing this out:

1) I think on `develop`, when you select a start date, the date picker doesn't allow you to pick an end date that's older, like the `minDate` is set to `startDate` or similar. I actually think I prefer how it works on this branch; BUT
2) When I change the start date/time, the end date/time *always* changes to 1 hour after start time. I think this is quite frustrating and people might not even notice, so I think the end date/time should only change in case you select a start datetime that's after the selected end datetime.

Reply:
@aapeliv I'm actually thinking maybe I should set these null by default too! 

But it's not always one hour, just that by default. Try this:

1. startTime 9am by default (endTime 10am)
2. Change endTime to 1pm
3. Change startTime to 10am, endTime will change to 2pm (keeping the same difference i.e. "dateDelta").

Do you think I should just remove all the "helping" with the dates and just keep the validation if it's after the startTime?

Reply:
@aapeliv Okay I pushed some changes like this. Let me know if you're happy with it and I'll update the tests.

@jesseallhands tagging you here as an FYI since this is a design/functionality change, but I think it kinda goes along with the birthdate change as well where we do null by default so they have to consciously pick. That prevents accidental wrong dates and times.

Reply:
> Do you think I should just remove all the "helping" with the dates

Since I implemented it and yet can't quite remember why we did it that way... I wonder if it's because Couchsurfing works that way? 🤔 Or are there well known event scheduling tools that give users this level of "help"? Google Calendar perhaps?

Reply:
@darrenvong I wish I knew haha I'm sure it was cool at the time!

What we found is people were just going with the defaults and then getting confused why it had those dates, especially the birthdate hah so trying to force them to think about it.